src/Resizer.cs

```diff
@@ -1,27 +1,58 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.IO;
-using ImageSharp;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats;
+using SixLabors.ImageSharp.Formats.Bmp;
+using SixLabors.ImageSharp.Formats.Gif;
+using SixLabors.ImageSharp.Formats.Jpeg;
+using SixLabors.ImageSharp.Formats.Png;
+using SixLabors.Primitives;
 
 namespace ImageResizer
 {
     public static class Resizer
     {
+        private static Image<Rgba32> _image;
+        private static IImageEncoder _encoder;
+
         public static void Resize(string filePath, IEnumerable<int> widths)
         {
             using (var stream = File.OpenRead(filePath))
             {
-                var image = Image.Load(stream);
-                var aspectRatio = new Size(image.Width, image.Height).AspectRatio();
+                _image = Image.Load(stream);
+
+                var filetype = Path.GetExtension(filePath);
+
+                switch (filetype)
```

Comment:
I noticed in Program.cs that you support tif and tiff files, but your docs say you only support the file types listed below.  If SixLabors.ImageSharp can handle tiff (it should), I'll update the file-type checking (which is necessary for the Encoder, not just that thrown error) and the ReadMe.